redaxo/src/core/lib/login/password_policy.php

```diff
@@ -63,6 +63,7 @@ public function getDescription(): ?string
      *
      * @deprecated since 5.12, use `getDescription` instead
      */
+    #[\JetBrains\PhpStorm\Deprecated( reason: 'since 5.12, use `getDescription` instead', replacement: '%class%->getDescription()')]
```

Comment:
```suggestion
    #[\JetBrains\PhpStorm\Deprecated(reason: 'since 5.12, use `getDescription` instead', replacement: '%class%->getDescription()')]
```